frontend/src/components/elements/BokehChart/BokehChart.tsx

```diff
@@ -35,6 +38,20 @@ class BokehChart extends React.PureComponent<Props> {
 
   private updateChart = (data: any): void => {
     const chart = document.getElementById(this.chartId)
+    const plot =
+      data && data.doc && data.doc.roots && data.doc.roots.references
+        ? data.doc.roots.references.find((e: any) => e.type === "Plot")
+        : undefined
+    if (plot) {
+      if (!plot.attributes.plot_width) {
```

Comment:
When does the plot not have a height or width?

Reply:
When you do not specify those values on chart creation, in your python script:
# create a new plot with a title and axis labels
p = figure(title="simple line example", x_axis_label="x", y_axis_label="y")
or
p = figure(title="simple line example", x_axis_label="x", y_axis_label="y", plot_width=200)